docs/usage/setup-azure-devops.md

```diff
@@ -3,74 +3,77 @@ title: Azure DevOps setup
-               cat > .npmrc << EOF
-               PLACEHOLDER(.NPMRC CONTENTS)
-               EOF
+   steps:
+     - task: npmAuthenticate@0
+       inputs:
+         workingFile: .npmrc
+   
+     - bash: |
+         git config --global user.email 'bot@renovateapp.com'
+         git config --global user.name 'Renovate Bot'
+         cp .npmrc ~/.npmrc
+         npx renovate
+       env:
+         TOKEN: $(System.AccessToken)
+   ```
+
+1. Add a file named `.npmrc` to your repository with the following contents:  
+   (replacing `YOUR-ORG` with your Azure DevOps organization and `YOUR-FEED` with your Azure Artifacts feed)
 
-         - task: npmAuthenticate@0
-           displayName: npm Authenticate
-           inputs:
-             workingFile: .npmrc
+   ```
+   @enpowerx:registry=https://pkgs.dev.azure.com/YOUR-ORG/_packaging/YOUR-FEED/npm/registry/ 
+   always-auth=true
+   ```
 
-         - task: Bash@3
-           displayName: Create renovate config
-           inputs:
-             targetType: inline
-             script: |
-               cat > config.js << EOF
-               module.exports = {
-                 platform: 'azure',
-                 endpoint: 'https://dev.azure.com/PLACEHOLDER(ORGANIZATION)/',
-                 token: '$(System.AccessToken)',
-                 npmrc:
-                   '$(sed ':a;N;$!ba;s/\n/\\n/g' .npmrc)',
-                 ignoreNpmrcFile: true,
-                 repositories: ['PLACEHOLDER(PROJECT)/PLACEHOLDER(REPO NAME)']
-               };
-               EOF
+1. Add a file named `config.js` to your repository with the following contents:  
+   (replacing `YOUR-ORG` with your Azure DevOps organization and `YOUR-PROJECT/YOUR-REPO` with your Azure DevOps project and repository)
 
-         - task: Bash@3
-           displayName: Run renovate
-           inputs:
-             targetType: inline
-             script: |
-               # Git credentials
-               git config --global user.email 'bot@renovateapp.com'
-               git config --global user.name  'Renovate Bot'
-               # Run renovate
-               npx --userconfig .npmrc renovate
+   ```javascript
+   module.exports = {
+     platform: 'azure',
+     endpoint: 'https://dev.azure.com/YOUR-ORG/',
+     token: process.env.TOKEN,
+     hostRules: [
+       {
+         hostName: 'pkgs.dev.azure.com',
+         username: 'apikey',
+         password: process.env.TOKEN,
+       }
+     ],
+     repositories: [
+       'YOUR-PROJECT/YOUR-REPO'
+     ]
+   };
    ```
 
 1. Additionally, you can add `renovate.json` with Renovate configurations in the root of the repo. [Read more about configurations options](https://docs.renovatebot.com/configuration-options/)
+
+If you wish to use a single Renovate pipeline to update multiple repositories:
+
+1. Add the names of the repositories to `config.js`.
+
+1. Make sure that the "Project Collection Build Service" user has the following permissions on the repositores:
```

Comment:
If I recall correctly, there are at least 2 users with this name: 1 at the org level, and 1 at the project level. They're differentiated with the org or project in brackets after the name i.e. "Project Collection Build Service (YOUR-PROJECT)"

Let me clarify this, and get back to you.

Reply:
You're right, the user is also named "Project Collection Build Service (YOUR-PROJECT)" in my setup, I just forgot to include it here. I've updated the PR.